code/modules/jobs/job_types/chaplain/chaplain_nullrod.dm

```diff
@@ -646,3 +690,171 @@
 	attack_verb_simple = list("stab", "poke", "slash", "clock")
 	hitsound = 'sound/weapons/bladeslice.ogg'
 	menu_description = "A pointy spear which penetrates armor a little. Can be worn only on the belt."
+
+// Nullblade - For when you really want to feel like rolling dice during combat
+
+/obj/item/nullrod/nullblade
+	name = "nullblade"
+	desc = "Clerical assassins are not officially recognized by the collective faiths of Nanotrasen. And yet, here you are."
+	icon = 'icons/obj/weapons/sword.dmi'
+	icon_state = "nullsword"
+	inhand_icon_state = "nullsword"
+	worn_icon_state = "nullsword"
+	lefthand_file = 'icons/mob/inhands/weapons/swords_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/weapons/swords_righthand.dmi'
+	w_class = WEIGHT_CLASS_BULKY
+	force = 12
+	wound_bonus = 10
+	bare_wound_bonus = 30
+	slot_flags = ITEM_SLOT_BELT
+	block_sound = 'sound/weapons/parry.ogg'
+	sharpness = SHARP_POINTY
+	hitsound = 'sound/weapons/bladeslice.ogg'
+	attack_verb_continuous = list("attacks", "punctures", "stabs", "slices", "tears", "lacerates", "rips", "dices", "cuts")
+	attack_verb_simple = list("attack", "puncture", "stab", "slice", "tear", "lacerate", "rip", "dice", "cut")
+	menu_description = "A blade that deals variable, low amounts of damage, but does easily inflict wounds. \
+		The stronger your swinging arm is, the stronger the blade is, though only slightly. \
+		Against debilitated targets, can also deal additional sneak attack damage with a very high wound chance."
+
+/obj/item/nullrod/nullblade/melee_attack_chain(mob/user, atom/target, params)
+	//Track our actual force separately
+	var/old_force = force
+	force = 0
+	//Potential dice roll for our baseline force
+	force += roll("1d6")
+
+	//Now we can check for our user's potential 'strength' value. As a baseline, we'll use a default value of 4 for the sake of nonhuman users.
+	var/strength_value = 4
+
+	//We can use our human wielder's arm strength to determine their 'strength'. We add unarmed lower and upper, then divide by four.
+	//This isn't how strength works in dnd but who fucking cares.
+	if(ishuman(user))
+		var/mob/living/carbon/human/human_user = user
+		var/obj/item/bodypart/wielding_bodypart = human_user.get_active_hand()
+		strength_value = round((wielding_bodypart.unarmed_damage_low + wielding_bodypart.unarmed_damage_high) * 0.25)
+
+	force += strength_value
+
+	//If our old_force is higher than our initial force, add the difference to this calculation.
+	//We do this because our force could have been changed by things like whetstones and RPG stats.
+	force += old_force - initial(force)
+
+	. = ..()
+	//Reapply our old force.
+	force = old_force
+
+/obj/item/nullrod/nullblade/afterattack(atom/target, mob/user, proximity, params)
+	. = ..()
+	if(!proximity)
+		return
```

Comment:
Afterattack PR was just merged so be sure to update this

Reply:
Sweet